src/main/java/androidx/graphics/Path.kt

```diff
@@ -116,3 +116,19 @@ inline infix fun Path.xor(p: Path): Path {
         op(p, Path.Op.XOR)
     }
 }
+
+/**
+ * Generate a closed shape from [points].
+ */
+@RequiresApi(19)
```

Comment:
What makes this 19+?

Reply:
Copied from method above, I'll remove it.